src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersDraggableGroup.tsx

```diff
@@ -0,0 +1,69 @@
+import styles from '@common/modules/table-tool/components/TableHeadersDraggableGroup.module.scss';
+import TableHeadersGroupControls from '@common/modules/table-tool/components/TableHeadersGroupControls';
+import TableHeadersReadOnlyList from '@common/modules/table-tool/components/TableHeadersReadOnlyList';
+import getTableHeaderGroupId from '@common/modules/table-tool/components/utils/getTableHeaderGroupId';
+import useTableHeadersContext from '@common/modules/table-tool/contexts/TableHeadersContext';
+import classNames from 'classnames';
+import React from 'react';
+import { Draggable } from 'react-beautiful-dnd';
+
+interface Props {
+  index: number;
+  legend: string;
+  name: string;
+  totalItems: number;
+  onMoveGroupToOtherAxis: () => void;
+}
+
+const TableHeadersDraggableGroup = ({
+  index,
+  legend,
+  name,
+  totalItems,
+  onMoveGroupToOtherAxis,
+}: Props) => {
+  const { groupDraggingActive } = useTableHeadersContext();
+  const groupId = getTableHeaderGroupId(legend);
+
+  return (
+    <>
+      <Draggable draggableId={name} index={index}>
+        {(draggableProvided, draggableSnapshot) => (
+          <div
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...draggableProvided.draggableProps}
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...draggableProvided.dragHandleProps}
+            className={classNames(styles.group, {
+              [styles.isDragging]: draggableSnapshot.isDragging,
+              [styles.isDraggingActive]:
+                !draggableSnapshot.isDragging && groupDraggingActive,
+              [styles.isDraggedOutside]:
+                draggableSnapshot.isDragging && !draggableSnapshot.draggingOver,
+            })}
+            ref={draggableProvided.innerRef}
+            role="button"
+            tabIndex={0}
+          >
+            <TableHeadersReadOnlyList
+              id={groupId}
+              legend={legend}
+              name={name}
+            />
+          </div>
+        )}
+      </Draggable>
+      {!groupDraggingActive && (
```

Comment:
The controls are removed because they're not in the draggable (as they would be nested buttons which causes accessibility problems) so get left behind when the groups move if they're not hidden. Agree it is a bit janky but I couldn't think of a better solution, any ideas?

Reply:
No, the buttons would need to be in the Draggable to be dragged with the group, which we can't have because nested buttons aren't allowed. Disabling them on drag wouldn't make any difference as they need to be active when nothing is being dragged, so causing the nested buttons problem.

Reply:
Yep, gave it a little play and I think we can do something like this :smile: 
```tsx
<Draggable>
  {(draggableProvided, draggableSnapshot) => (
    <div
      {...draggableProvided.draggableProps}
      ref={draggableProvided.innerRef}
    >
      <div 
        {...draggableProvided.dragHandleProps}
      >
      </div>
      <TableHeadersGroupControls ... />
    </div>
  )}
</Draggable>
```

Should give us this:

![image](https://user-images.githubusercontent.com/9917868/186180175-3b28a538-625e-4c63-8a94-ab7fa2ccf800.png)

Reply:
That totally works, I'm kicking myself for not thinking of it! I've moved the focus to be around the whole group as well.